tutoroars/templates/oars/jobs/init/superset-add-admin.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -e
+
+#
+# Always install local overrides first
+#
+/usr/bin/env bash /app/docker/docker-bootstrap.sh
+
+# Create an admin user
+echo "Setting up admin user"
+superset fab create-admin \
+  --username "{{ SUPERSET_ADMIN_USERNAME }}" \
+  --password "{{ SUPERSET_ADMIN_PASSWORD }}" \
+  --firstname Superset \
+  --lastname Admin \
+  --email "{{ SUPERSET_ADMIN_EMAIL }}"
+echo "Setting up admin user...Complete"
+
+# Update the password of the Admin user
+# (in case it changed since the user was created)
+echo "Resetting password for admin user"
+superset fab reset-password \
+  --username "{{ SUPERSET_ADMIN_USERNAME }}" \
+  --password "{{ SUPERSET_ADMIN_PASSWORD }}"
+echo "Resetting password for admin user...Complete"
```

Comment:
Do https://github.com/openedx/tutor-contrib-superset/pull/6/files#diff-5749faaf28779d5be58a5c45a42924f058c0ce27d6191054130e059c881fe81e belong here too?

Reply:
My gut is that the security features to work with the LMS belong here, but the xapi task belongs in the OARS repo, since I think it's possible people could want to use superset against the MySQL tables without xAPI. Does that sound right to you?